<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMainWindow</class>
 <widget class="QMainWindow" name="XMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blue Berry Transport Usig Dijkstra</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFarm"/>
   <addaction name="actionShop"/>
   <addaction name="actionMid"/>
   <addaction name="actionPath"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionShow"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFarm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Farm</string>
   </property>
  </action>
  <action name="actionShop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shop</string>
   </property>
  </action>
  <action name="actionMid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mid</string>
   </property>
  </action>
  <action name="actionPath">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Path</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Result</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>XMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
